src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -367,10 +368,9 @@ public sealed override bool Succeeded
 
         private static ImmutableArray<ISymbol> Normalize(IEnumerable<Symbol> data)
         {
-            var builder = ArrayBuilder<Symbol>.GetInstance();
-            builder.AddRange(data.Where(s => s.CanBeReferencedByName));
-            builder.Sort(LexicalOrderSymbolComparer.Instance);
-            return builder.ToImmutableAndFree().As<ISymbol>();
+            var builder = ArrayBuilder<ISymbol>.GetInstance();
+            builder.AddRange(data.Where(s => s.CanBeReferencedByName).OrderBy(s => s, LexicalOrderSymbolComparer.Instance).GetPublicSymbols());
```

Comment:
>builder.AddRange [](start = 12, length = 16)

The builder is no longer providing much value. Consider using `ImmutableArray.CreateRange(data.Where(...).OrderBy(...).GetPublicSymbols())` directly.